pkg/api/helper/helpers.go

```diff
@@ -602,3 +603,23 @@ func StorageNodeAffinityToAlphaAnnotation(annotations map[string]string, affinit
 	annotations[api.AlphaStorageNodeAffinityAnnotation] = string(json)
 	return nil
 }
+
+type MountPropagationMap map[string]map[string]api.MountPropagation
+
+func GetPodPropagation(podAnnotations map[string]string) (MountPropagationMap, error) {
```

Comment:
Add godoc for exported function?  I think some lint tools will complain about this...